e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/remove-fees-from-payment.spec.js

```diff
@@ -161,12 +161,12 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can remove fees from payments`, () =>
     cy.get('strong[data-cy="fee-record-status"]:contains("MATCH")').should('exist');
     pages.utilisationReportPage.premiumPaymentsTab.clickPaymentLink(paymentId);
 
-    cy.url().should('eq', relative(`/utilisation-reports/${reportId}/edit-payment/${paymentId}`));
+    cy.url().should('eq', relative(`/utilisation-reports/${reportId}/edit-payment/${paymentId}?redirectTab=premium-payments`));
 
     getFeeRecordCheckbox(1).check();
     pages.utilisationReportEditPaymentPage.clickRemoveSelectedPaymentsButton();
 
-    cy.url().should('eq', relative(`/utilisation-reports/${reportId}/edit-payment/${paymentId}`));
+    cy.url().should('eq', relative(`/utilisation-reports/${reportId}/edit-payment/${paymentId}?redirectTab=premium-payments`));
```

Comment:
@sw-mattw this assertion is failing because I haven't had time to add the logic to retain the redirect tab in the "remove selected fees" journey - once that is added, this test should pass